src/presentation/controller/addressLookUp/Controller.ts

```diff
@@ -423,12 +429,17 @@ class AddressLookUpController extends AbstractController {
     response.redirect(pageRouting.nextUrl);
   }
 
+  // TODO As each pageType can have a cache key, add the key to each 'routing' object for each page and remove the if/elses
```

Comment:
added this todo as having a long if/else is not good.

Reply:
Nice.